BeVolt/Tests/Test_BSP_CAN.c

```diff
@@ -0,0 +1,14 @@
+#include "common.h"
+#include "config.h"
+#include "BSP_CAN.h"
+
+ int main(void){
+   uint8_t message[8] = {0x01, 0x82,0x00,0x36,0x75,0,0,0x96};
+   uint8_t readData[8] = {0,0,0,0,0,0,0,0};
+   uint32_t id[1];
+  
+   uint8_t length = 8;
+   
+   BSP_CAN_Write(0x10A, message, length);
+   BSP_CAN_Read(id, readData);
+}
```

Comment:
Good test, but I would also test it using the CLI. There is functionality there to read CAN messages and also to write a CAN message to a specific id. So I'd just use that as well to test and make sure the BSP code works properly.